src/analytics.py

```diff
@@ -0,0 +1,59 @@
+from...import utils
```

Comment:
You have `...` here, try replacing with `import utils`.  Also - please ask questions in the discussion forum and/or open a pull request when you find you are spinning your wheels.